agent/php_execute.c

```diff
@@ -992,16 +991,16 @@ static void nr_php_execute_metadata_add_code_level_metrics(
   const char* filepath = NULL;
   const char* namespace = NULL;
   const char* function = NULL;
-  uint32_t lineno = 1;
-
   /*
    * Check if code level metrics are enabled in the ini.
    * If they aren't, exit and don't update CLM.
    */
+#if ZEND_MODULE_API_NO < ZEND_8_0_X_API_NO \
+    || defined OVERWRITE_ZEND_EXECUTE_DATA
   if (!NRINI(code_level_metrics_enabled)) {
```

Comment:
`code_level_metrics_enabled` does not seem to have a section in the ini template yet- is this being added in a separate PR?

Reply:
Yes, it was added in this PR for oapi:https://github.com/newrelic/newrelic-php-agent/pull/506/files#diff-07c39860ed983532dfdc2a9d3093475beb7bd749067607c324a1e7718cffd92fR2870

It's also added in this PR on dev: https://github.com/newrelic/newrelic-php-agent/pull/585/files#diff-07c39860ed983532dfdc2a9d3093475beb7bd749067607c324a1e7718cffd92fR3108